tools/docker/Dockerfile.integ-test

```diff
@@ -40,6 +45,18 @@ RUN containerd 2>/dev/null & \
   ctr --address /run/firecracker-containerd/containerd.sock content fetch docker.io/library/alpine:3.10.1 >/dev/null && \
   ctr --address /run/firecracker-containerd/containerd.sock content fetch docker.io/mlabbe/iperf3:3.6-r0 >/dev/null
 
+# Install crictl and critest
+ENV VERSION="v1.23.0"
+RUN wget https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/critest-$VERSION-linux-amd64.tar.gz
+RUN tar zxvf critest-$VERSION-linux-amd64.tar.gz -C /tmp/
+RUN install -D -o root -g root -m755 -t /usr/local/bin /tmp/critest
+RUN rm -f critest-$VERSION-linux-amd64.tar.gz
+
+RUN wget https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-$VERSION-linux-amd64.tar.gz
+RUN tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /tmp/
+RUN install -D -o root -g root -m755 -t /usr/local/bin /tmp/crictl
+RUN rm -f crictl-$VERSION-linux-amd64.tar.gz
```

Comment:
Let's put them in single RUN. https://hackernoon.com/tips-to-reduce-docker-image-sizes-876095da3b34